target/linux/imx6/base-files/lib/upgrade/platform.sh

```diff
@@ -60,7 +60,9 @@ platform_check_image() {
 	gw,imx6q-gw5907 |\
 	gw,imx6q-gw5910 |\
 	gw,imx6q-gw5912 |\
-	gw,imx6q-gw5913 )
+	gw,imx6q-gw5913 |\
+	xiaomi,dgnwg05lm |\
+	xiaomi,gateway-lumi )
```

Comment:
This is only one device, so stick to the name. Same below.

Reply:
Left only` xiaomi,gateway-lumi` as the generic name for xiaomi/aqara devices